Altis_Life.Altis/core/functions/fn_keyHandler.sqf

```diff
@@ -143,8 +143,8 @@ switch (_code) do {
 
     //T Key (Trunk)
     case 20: {
-        if (!_alt && !_ctrlKey && !dialog && {!life_action_inUse}) then {
-            if (!(isNull objectParent player) && alive vehicle player) then {
+        if (!_alt && !_ctrlKey && !dialog &&{!life_action_inUse}) then {
+            if (!(isNull objectParent player) && alive vehicle player && (!(player getVariable ["restrained",false])) && (!(player getVariable ["playerSurrender",false])) && !life_isknocked && !life_istazed) then {
```

Comment:
Hello @BoGuu
I tried but I did not find how to do it, can you help me and explain to me for the next time